src/main/java/com/microsoft/azure/functions/worker/broker/JavaFunctionBroker.java

```diff
@@ -130,7 +130,6 @@ void addDirectory(File directory) throws IOException {
 	}
 
 	public void addJavaAnnotationLibrary() throws IOException {
-		WorkerLogManager.getSystemLogger().warning("Customer is not providing java annotation files, they may not use the latest version of java library and maven plugin.");
```

Comment:
as we are catch-all, this warning will be logged whether cx provide java library or not. So it's unnecessary here.

Reply:
Can we log this somewhere else. It would help understand what if customer is using the latest java library and maven plugin

Reply:
Will create a issue for tracking this, it requires add some logics to check if cx is adding library themselves and what version of library they are using.

Reply:
issue opened https://github.com/Azure/azure-functions-java-worker/issues/575